@xen-orchestra/xapi/vm.js

```diff
@@ -45,6 +45,21 @@ const cleanBiosStrings = biosStrings => {
   }
 }
 
+async function listNobakVbds(xapi, vbdRefs) {
```

Comment:
`this` is not defined because it's not a method.

It's kind of a private method, but due to the weird implementation of this lib, it's defined outside.